src/main/java/cinema/controllers/CustomGlobalExceptionHandler.java

```diff
@@ -0,0 +1,33 @@
+package cinema.controllers;
+
+import java.util.Date;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+@ControllerAdvice
+public class CustomGlobalExceptionHandler extends ResponseEntityExceptionHandler {
+    @Override
+    protected ResponseEntity<Object> handleMethodArgumentNotValid(
```

Comment:
```suggestion
    protected ResponseEntity<BaseResponse> handleMethodArgumentNotValid(
```
Краще параметризувати якимось класом ніж Object. Всередині твого класу BaseResponse як поле `errors` може бути та ж сама мапа, яку ти реалізувала

Reply:
та я что-то не могу, все методы @override возвращают только ResponseEntity<Object>

Reply:
Ти і повинна повертати ResponseEntity, Бодя мав на увазі параметризвувати його не Object а кастомним класом